.github/workflows/dependent-issues.yml

```
@@ -0,0 +1,42 @@
+---
+name: Dependent Issues
+
+on:
+  issues:
+    types:
+      - opened
+      - edited
+      - reopened
+  pull_request_target:
+    types:
+      - opened
+      - edited
+      - reopened
+      - synchronize
+  pull_request:
+    types:
+      - opened
+      - edited
+      - reopened
+  schedule:
+    - cron: '0 * * * *' # check hourly
+
+jobs:
+  check:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: z0al/dependent-issues@v1
+        env:
+          # (Required) The token to use to make API calls to GitHub.
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          # (Optional) The label to use to mark dependent issues
+          label: dependent
+
+          # (Optional) Enable checking for dependencies in issues. Enable by
+          # setting the value to "on". Default "off"
```

```suggestion
          # Enable checking for dependencies in issues. 
```

Reading this and looking at the main docs, I'm still not sure what this does. 😅